src/util.h

```diff
@@ -67,6 +67,14 @@ extern "C" {
 
 #define COMPILE_ERROR_ON(cond) ((void)sizeof(char[(cond) ? -1 : 1]))
 
+#define pf_always_inline __attribute__((always_inline)) inline
```

Comment:
They are not needed now. If we'll ever port pmemfile to MSVC these will be needed. And I think having wrappers around compiler-specific code like attributes is a "good thing" (tm).